BLE_GattServer_ExperimentalServices/source/main.cpp

```diff
@@ -0,0 +1,179 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2015 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ble/BLE.h"
+#include "ble/gap/Gap.h"
+#include "mbed-os-ble-utils/pretty_printer.h"
+#include "mbed-os-experimental-ble-services/services/inc/LinkLossService.h"
```

Comment:
```suggestion
#include "pretty_printer.h"
#include "services/inc/LinkLossService.h"
```
for new cmake compatibility

Reply:
still needs fixing